### PR TITLE
Updated ThemeSwitcher component: className exposed + new icons

### PR DESCRIPTION
**Pull request by cheina97**
Components updated:

- ThemeSwitcher (common)

This PR adds the className prop, in order to expose the internal switch's className and 2 new icons (sun/moon).

### CODE REVIEW

**Review by giorio94 — APPROVED** [review]

File: frontend/src/components/misc/ThemeSwitcher/ThemeSwitcher.tsx [hunk 1]
```diff
@@ -1,20 +1,36 @@
...
-  return (
+  return buttonMode ? (
+    <Button
+      onClick={onChange}
+      className={'themeswitcher-button ' + className}
+      size="large"
+      shape="round"
+      type="text"
+    >
+      {isDarkTheme ? 'Light theme' : 'Dark theme'}
```
Comment (QcFe) [inline frontend/src/components/misc/ThemeSwitcher/ThemeSwitcher.tsx:29]:
```suggestion
      {isDarkTheme ? 'Light' : 'Dark'} theme
```

**Review by QcFe — APPROVED** [review]
Just a tiny thing

**Pull request merged**